### ISSUE TITLE
legend_position breaks in hv 1.11.0

### ISSUE THREAD

**Issue by ea42gh** [pyviz/holoviews#3361]
```
def show_another_holomap(sz):
    def ith(i):
        x_vals = np.linspace(0,sz,100)
        h1 = hv.Curve( (x_vals,2.*i+sz*np.random.uniform(-2,2,size=100)), label='C1')
        h2 = hv.Curve( (x_vals,sz+  np.random.uniform(-2,2,size=100)), label='C2')
        return hv.Overlay([h1,h2]).relabel('Sz = %4.1f'%sz).opts(legend_position='top')
    return hv.HoloMap({ i: ith(i) for i in range(5)}, kdims=['Plot_Number'])

(show_another_holomap(.1)+\
 show_another_holomap(5.)).opts( {"Curve": dict(axiswise=True, framewise=True)} )
```
fails

**Comment by philippjfr** [comment]
I believe this is a regression in bokeh, looking at the JS console I get:

```
Uncaught Error: unsatisfiable constraint: SidePanel(413F53C6A101488EBF9177BFEE97D38A, above).height - 60 >= 0
```

**Comment by philippjfr** [comment]
Seems fixed upstream now, so once bokeh 1.1 is out this should be fixed. Closing.
